src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -132,7 +131,7 @@ goog.require('ga_permalink');
                     elevation_model: 'COMB'
                   }
                 }).success(function(response) {
-                  scope.altitude = parseInt(response.height);
+                  scope.altitude = parseFloat(response.height);
```

Comment:
https://github.com/geoadmin/mf-geoadmin3/pull/2205 and https://github.com/geoadmin/mf-geoadmin3/issues/2185. Was the decision reversed during this work?